gatsby-node.js

```diff
@@ -34,12 +34,16 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     reporter.panicOnBuild('Error while running GraphQL query.');
     return;
   }
-
-  result.data.allMarkdownRemark.edges.forEach(({ node }) => {
+  const posts = result.data.allMarkdownRemark.edges;
+  result.data.allMarkdownRemark.edges.forEach(({ node }, index) => {
```

Comment:
This should work, sorry, missed this earlier

```suggestion
  posts.forEach(({ node }, index) => {
```

Reply:
Done